OnlineFlightSearchAPI/Program.cs

```diff
@@ -15,6 +18,9 @@ public class Program
         public static void Main(string[] args)
         {
             CreateWebHostBuilder(args).Build().Run();
+
+            IUnityContainer container = new UnityContainer();
```

Comment:
Asp.net has a built-in container
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-2.1